descriptors/sqlserver/debezium-sqlserver-1.9.0.Alpha1.json

```diff
@@ -71,35 +71,6 @@
           "x-name" : "database.dbname",
           "x-category" : "CONNECTION"
         },
-        "database.names" : {
```

Comment:
@ani-sha I think this must be included.

Reply:
@ani-sha do you have notes about the outcome? Added you the decisions to the meeting notes?

Reply:
database.names is part of the currently being worked-on multiple DB support for this connector. It's not ready for prime-time yet.